.scalafix.conf

```diff
@@ -0,0 +1,17 @@
+rules = [
+  ExplicitResultTypes
+  NoAutoTupling
+  NoValInForComprehension
+  OrganizeImports
+  ProcedureSyntax
+  RemoveUnused
+]
+
+OrganizeImports {
+  # Align with IntelliJ IDEA so that they don't fight each other
+  groupedImports = Merge
```

Comment:
After dealing with a couple merge conflicts: can we _not_ do grouped imports? It's much easier on diffs to keep each on a separate line. If it becomes too much you can always use the wildcard.